tablet-report-parser/moses.pl

```diff
@@ -805,14 +805,22 @@ sub Create_Views{
     SELECT '$0' as program, '$VERSION' as version, '$opt{STARTTIME_PRINTABLE}' AS run_on, '$opt{LOCALHOST}' as host;
 
   CREATE VIEW tablets_per_node AS 
-    SELECT node_uuid,min(public_ip) as node_ip,min(region ) as region,  count(*) as tablet_count,
-           sum(CASE WHEN private_ip = leader THEN 1 ELSE 0 END) as leaders,
+    SELECT node_uuid,ip as node_ip,zone,  count(*) as tablet_count,
+           sum(CASE WHEN status='TABLET_DATA_COPYING' THEN 1 ELSE 0 END) as copying,
            sum(CASE WHEN tablet.state = 'TABLET_DATA_TOMBSTONED' THEN 1 ELSE 0 END) as tombstoned,
+           sum(CASE WHEN node_uuid = leader THEN 1 ELSE 0 END) as leaders,
```

Comment:
This changed from `private_ip = leader` to `node_uuid = leader`. Did the definition of `leader` change? If so, where?

Reply:
Thanks for catching this - this was a mistake - it is being reverted, and corrected.